db/migration/20210329121921_create_shares_table.go

```diff
@@ -0,0 +1,34 @@
+package migrations
```

Comment:
This file may need to be renamed to a more recent date, or else it won't be applied as thee are newer migrations

Reply:
Done.